modules/avatar/react/stories/stories_visualTesting.tsx

```diff
@@ -0,0 +1,93 @@
+/// <reference path="../../../../typings.d.ts" />
+/** @jsx jsx */
+import {jsx} from '@emotion/core';
+import {StaticStates} from '@workday/canvas-kit-labs-react-core';
+import {
+  ComponentStatesTable,
+  permutateProps,
+  withSnapshotsEnabled,
+} from '../../../../utils/storybook';
+import {Avatar, AvatarButton} from '../index';
+
+// @ts-ignore: Cannot find module error
+import testAvatar from './test-avatar.png';
+// eslint-disable-next-line no-empty-function
+const noop = () => {};
+
+export default withSnapshotsEnabled({
+  title: 'Testing|React/Indicators/Avatar',
+  component: Avatar,
+  parameters: {
+    chromatic: {
+      delay: 300, // Ensure we capture the image after loading and transition
```

Comment:
NB: I could see this being a utils as well since we have a few components that have transitions

Reply:
Most of the time we turn off transitions w/ `transformOrigin={null}` rather than using a delay though. This is the only place we use a chromatic delay atm so going to leave as is for now